portal-web/test/functional/com/liferay/portalweb/macros/analyticscloud/ACSites.macro

```diff
@@ -1,5 +1,15 @@
 definition {
 
+	macro getCurrentMetric {
+		WaitForVisible(
+			key_cardTitle = "Visitors Behavior",
+			locator1 = "ACCards#CARD_TITLE");
+
+		var current = selenium.getText("//span[@class='title'and text()='${metricName}']/..//div[@class='metric-value']");
```

Comment:
Hey @joshitagaki, very nice work here. I think you could create a path to not use the path directly in the `.macro`. What do you think?

Reply:
Thanks! Yea I moved it to .path file using an unused path locator